src/components/radioButtonCO.js

```diff
@@ -10,7 +10,7 @@ import palette from "../theme/palette"
 const useStyles = makeStyles({
```

Comment:
```suggestion
const useStyles = makeStyles((theme) => ({
```
Remove the imports **theme** and **palette** located at the top.